047-TrafficControlWithDapr/README.md

```diff
@@ -53,16 +56,19 @@ When the car passes an exit-camera, another photo and timestamp are registered.
   - Linux ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt))
   - macOS ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-macos))
   - Windows ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli))
-- Install Bicep extension for VS Code ([instructions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep))   
+- Install Bicep extension for VS Code ([instructions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep))
 - If you're running Windows, you'll need to install a **bash shell** to run some of the commands. Install either the [Git Bash](https://git-scm.com/downloads) client or the [Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+- Helm ([instructions](https://helm.sh/docs/intro/install/))
 
 ## Repository Contents
```

Comment:
Remove this section as it should be moved to the Coach guide's README.md as per the new WTH templates.

Reply:
done, updated Coach/README.md to follow new format & moved instructions around

Reply:
Doh. I think you got this comment wrong (or I wasn't clear).  The "Respository Contents" should be removed from the hack home page as they are now included in the Coach Guide ToC.

The Pre-req section that you had here IS okay to have on the home page.  Although... since we're slinging things around, that pre-req list belongs best in Challenge 0.

Thanks for adding the extra sections into the Coach guide from the template!

Reply:
@jrzyshr, ok, moved the Repository Contents section